From 6557b499dc631761f296ea5cf5d1d0a8cd762443 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Tue, 3 Nov 2015 14:47:12 +0200
Subject: [PATCH 0459/1240] Revert "ddr: Update DDR bus width to 64-bit"

This reverts commit 064ab7d3dc4c92b07d182b5ece382cfaedf9c1c3.

	Reverted due to 64-bit DDR instability

Conflicts:
	drivers/ddr/ddr_mckinley6.c

Change-Id: I6b2bf5eafe1638f4003801c63c138911e07c1b39
Reviewed-on: http://vgitil04.il.marvell.com:8080/24544
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/ddr/ddr_mckinley6.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/ddr/ddr_mckinley6.c b/drivers/ddr/ddr_mckinley6.c
index ba1669d..0c69432 100644
--- a/drivers/ddr/ddr_mckinley6.c
+++ b/drivers/ddr/ddr_mckinley6.c
@@ -50,7 +50,7 @@ struct mvebu_mckinley_config mckinley_mac_config[] = {
 	{ 0x038, { 0x00000000} },
 	{ 0x03c, { 0x00000000} },
 	{ 0x040, { 0x22222022} },
-	{ 0x044, { 0x00030400} },
+	{ 0x044, { 0x00030300} },
 	{ 0x048, { 0x00000001} },
 	{ 0x04c, { 0x00000000} },
 	{ 0x050, { 0x00000000} },
@@ -900,7 +900,7 @@ struct mvebu_mckinley_config mckinley_phy_config[] = {
 	{ 0x200, { 0x00110001} },
 	{ 0x204, { 0x00000000} },
 	{ 0x220, { 0x11010642} },
-	{ 0x044, { 0x00030400} },
+	{ 0x44 , { 0x00030300} },
 	{ 0x2c0, { 0x0000e000} },
 	{ 0x2c4, { 0x00000020} },
 	{ 0x300, { 0x00000709} },
-- 
1.9.1

